src/main/java/core/basesyntax/figure/RightTriangle.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax.figure;
+
+public class RightTriangle extends AbstractFigure implements Figure {
```

Comment:
The class RightTriangle should not begin with an empty line according to the checklist.